tests/e2e_regression/e2e_config_2.yml

```diff
@@ -9,16 +9,20 @@ analysis:
     ipfs:
       gateway: https://ipfs.io
     fast_startup: true
+  helpers:
+    caching_proxies:
+      - { target: "https://sourcify.dev/server", host_addr: "localhost:9191" }
   analyzers:
-    evm_tokens_emerald:         { stop_on_empty_queue: true }
-    evm_nfts_emerald:           { stop_on_empty_queue: true }
-    evm_token_balances_emerald: { stop_on_empty_queue: true }
-    evm_contract_code_emerald:  { stop_on_empty_queue: true }
-    evm_abi_emerald:            { stop_on_empty_queue: true }
+    evm_tokens_emerald:            { stop_if_queue_empty_for: 1s }
+    evm_nfts_emerald:              { stop_if_queue_empty_for: 1s }
+    evm_token_balances_emerald:    { stop_if_queue_empty_for: 1s }
+    evm_contract_code_emerald:     { stop_if_queue_empty_for: 1s }
+    evm_abi_emerald:               { stop_if_queue_empty_for: 10s }
```

Comment:
Mind adding a comment here about how we expect to fetch all the abis within 10s, and that we may need to tune this in the future? (Alternatively we could add a third step via an `e2e_config_3.yml` but that seems overkill for now)

Reply:
+1 on both points about `_3.yml`. Comment added!